packages/sync/src/modules/Comments.php

```diff
@@ -177,6 +178,23 @@ public function get_whitelisted_comment_types() {
 		);
 	}
 
+	/**
+	 * Prevents any comment types that are not in the white list from being enqued and send to WordPress.com.
```

Comment:
Couple of typos here.

```suggestion
	 * Prevents any comment types that are not in the whitelist from being enqueued and sent to WordPress.com.
```